app/controllers/twitter_sign_in.js

```diff
@@ -4,7 +4,7 @@ var oauth = require('../../lib/oauth.js')
 var logger = require('../../lib/logger.js')()
 
 var oauthRequestTokenHandler = function (req, res) {
-  return function (err, oauth_token, oauth_token_secret, results) {
```

Comment:
What's the reason behind these changes from underscores between words to camel case? Standard style doesn't complain about them.